routes/web.php

```diff
@@ -158,106 +167,108 @@
         });
 
         Route::get('/', [SettingsController::class, 'renderSettings'])
-             ->name('settings');
+            ->name('settings');
         Route::post('/', [SettingsController::class, 'updateMainSettings']);
         Route::post('/update/privacy', [SettingsController::class, 'updatePrivacySettings'])
-             ->name('settings.privacy');
+            ->name('settings.privacy');
 
         Route::post('/password', [SettingsController::class, 'updatePassword'])
-             ->name('password.change');
+            ->name('password.change');
 
         Route::get('/follower', [\App\Http\Controllers\SettingsController::class, 'renderFollowerSettings'])
-             ->name('settings.follower');
+            ->name('settings.follower');
         Route::post('/follower/remove', [\App\Http\Controllers\SettingsController::class, 'removeFollower'])
-             ->name('settings.follower.remove');
+            ->name('settings.follower.remove');
         Route::post('/follower/approve', [SettingsController::class, 'approveFollower'])
-             ->name('settings.follower.approve');
+            ->name('settings.follower.approve');
         Route::post('/follower/reject', [SettingsController::class, 'rejectFollower'])
-             ->name('settings.follower.reject');
+            ->name('settings.follower.reject');
 
         Route::post('/uploadProfileImage', [FrontendUserController::class, 'updateProfilePicture'])
-             ->name('settings.upload-image');
+            ->name('settings.upload-image');
         Route::get('/deleteProfilePicture', [UserController::class, 'deleteProfilePicture'])
-             ->name('settings.delete-profile-picture');
+            ->name('settings.delete-profile-picture');
 
         Route::post('/delsession', [UserController::class, 'deleteSession'])
-             ->name('delsession');
+            ->name('delsession');
         Route::post('/deltoken', [UserController::class, 'deleteToken'])
-             ->name('deltoken');
+            ->name('deltoken');
+        Route::post('/delwebhook', [WebhookController::class, 'deleteWebhook'])
+            ->name('delwebhook');
```

Comment:
Are all the changes in this file, apart from the `delwebhook` route necessary? Would like to clean up..

Reply:
No, that was my formatter going crazy, sorry for that